specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json

```diff
@@ -103,6 +103,42 @@
         }
       }
     },
+    "/providers/Microsoft.ResourceHealth/emergingIssues/{issueName}": {
```

Comment:
To satisfy the RPC and allow people to discover the singleton there should also be a GET "/providers/Microsoft.ResourceHealth/emergingIssues" that returns a "value": [] array with the single "default" instance inside it with operationId "EmergingIssues_List"

Reply:
OK. done